security/fileCertManager/fileCertManager.go

```diff
@@ -0,0 +1,122 @@
+package fileCertManager
+
+import (
+	"crypto/tls"
+	"crypto/x509"
+	"github.com/fsnotify/fsnotify"
+	"github.com/go-ocf/kit/security"
+	"io/ioutil"
+	"strings"
+)
+
+// FileCertManager holds certificates from filesystem watched for changes
+type FileCertManager struct {
+	tlsKey []byte
+	tlsCert []byte
+	tlsKeyPair tls.Certificate
+	caAuthorities *x509.CertPool
+	watcher *fsnotify.Watcher
+	tlsKeyFileName string
+	dirPath string
+	tlsCertFileName string
+}
+
+// CloseManager ends watching certificates
+func (a *FileCertManager) CloseManager() {
+	_ = a.watcher.Close()
```

Comment:
pls wait for end go routine `watchFiles` and then a.watcher.Close() - synchronize

Reply:
Consider it done